### PR TITLE
Add a test to verify that an active attrib's base type

### PR DESCRIPTION
**Pull request by zhenyao**
needs to match the function type that sets its value.

This is a WebGL2 specific behavior.
See https://github.com/KhronosGroup/WebGL/pull/1861

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell Please review.

**Comment by kenrussell** [comment]
LGTM - but per comment on #1861 , VertexAttribPointer and VertexAttribIPointer should be tested too. (These are more commonly used than the single-value initialization functions.)

**Comment by zhenyao** [comment]
I updated the test to include the vertexAttrib{I}Pointer cases.  Please take another look.

File: sdk/tests/conformance2/rendering/attrib-type-match.html [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+                                  -1, -2,
+                                  -1, -2,
+                                  -1, -2,
+                                  -1, -2,
+                                  -1, -2]);
+    gl.bufferData(gl.ARRAY_BUFFER, offsetI, gl.STATIC_DRAW);
+
+    var offsetUBuffer = gl.createBuffer();
+    gl.bindBuffer(gl.ARRAY_BUFFER, offsetUBuffer);
+    var offsetU = new Int32Array([1, 2,
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/attrib-type-match.html]: This could be Uint32Array - not a big deal.

Comment (zhenyao) [inline sdk/tests/conformance2/rendering/attrib-type-match.html]: Right.  It's a typo bug. I'll fix it.

**Comment by kenrussell** [comment]
The new tests LGTM. Thanks for including them. One minor comment. Please feel free to merge yourself.

**Pull request merged**